crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -60,6 +60,17 @@ impl From<anyhow::Error> for SubmitApiError {
     }
 }
 
+#[derive(Debug)]
+pub enum SubmissionLoopStatus {
+    Enabled,
+    Disabled(DisabledReason),
+}
+
+#[derive(Debug)]
+pub enum DisabledReason {
+    CustomNodesDisabledMevExtractable,
```

Comment:
I think `MevExtractable` would be a better name here. It's an implementation detail that only `CustomNodes` suffers from this problem and `Disabled` is already in the enum name. Also all submission strategies have access to this value so it would theoretically be possible that an eden submitter has `CustomNodesDisabledMevExtractable` as its status which would be confusing.

Reply:
Yes, `MevExtractable` seems clearer.